### PR TITLE
Feat: Refinements on `Navbar` and `Navlinks`

### PR DESCRIPTION
**Pull request by renatamottam**
## What's the purpose of this pull request?
This PR follows new styles guidelines for `Navbar` and `Navlinks`

## How does it work?
Additionally to visual changes, we added a scroll interaction on desktop:

https://user-images.githubusercontent.com/11613011/182395277-1ff46d43-241d-4ef7-853c-905e7846b114.mov

## How to test it?
You can check the [Navbar](https://nextjs-store-storybook-git-feat-refinements-he-78a5d3-faststore.vercel.app/?path=/docs/organisms-navbar-overview--overview-default) and [Navlink](https://nextjs-store-storybook-git-feat-refinements-he-78a5d3-faststore.vercel.app/?path=/docs/organisms-navbar-navlinks--overview-default) stories.  You can also check the scroll effect at the [Demo Store](https://preview-191--nextjs.preview.vtex.app/).

## References
https://www.codemzy.com/blog/react-sticky-header-disappear-scroll

## Checklist
**PR Title and Commit Messages**
- [x] PR title follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification
- [x] Commit messages follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification

**PR Description**
- [x] Added a label according to the PR goal - `Breaking change`, `Enhancement`, `Bug` or `Chore`.
- [x] Added the component, hook, or pathname in-between backticks (``) *- If applicable*. E.g., *`ComponentName` component*.
- [x] Identified the function or parameter in the PR *- If applicable*. E.g., *`useWindowDimensions` hook*.

**Documentation**
- [x] PR description
- [x] Added to/Updated the Storybook - *if applicable*.

### CODE REVIEW

File: src/styles/global/tokens.scss [hunk 1]
```diff
@@ -198,6 +198,12 @@ body {
     --fs-grid-padding                  : var(--fs-spacing-5);
   }
 
+  // Z INDEX
```
Comment (hellofanny) [inline src/styles/global/tokens.scss:201]:
```suggestion
  // Z-INDEX
```

File: src/components/common/Navbar/navbar.module.scss [hunk 2]
```diff
@@ -161,24 +179,20 @@
...
-    padding-bottom: var(--fs-spacing-2);
+    padding: var(--fs-navbar-header-padding);
     overflow-x: clip;
-
-    @include media(">=notebook") {
-      padding-top: var(--fs-spacing-1);
-      padding-bottom: var(--fs-spacing-1);
-    }
+    background-color: var(--fs-navbar-bkg-color);
+    backdrop-filter: blur(10px);
```
Comment (hellofanny) [inline src/components/common/Navbar/navbar.module.scss:185]: 🆒

File: src/components/common/Navbar/navlinks.module.scss [hunk 3]
```diff
@@ -5,7 +5,14 @@
   // Design Tokens for Navlinks
   // --------------------------------------------------------
 
+  --fs-navlinks-bkg-color                        : var(--fs-color-body-bkg);
+
   --fs-navlinks-padding-top-notebook             : var(--fs-spacing-1);
+  --fs-navlinks-padding-bottom-notebook          : var(--fs-navlinks-padding-top-notebook);
+
```
Comment (hellofanny) [inline src/components/common/Navbar/navlinks.module.scss:12]:
```suggestion
  --fs-navlinks-padding-top-notebook             : var(--fs-spacing-1);
  --fs-navlinks-padding-bottom-notebook          : var(--fs-navlinks-padding-top-notebook);

  --fs-navlinks-bkg-color                        : var(--fs-color-body-bkg);

```

File: src/components/common/Navbar/Navbar.tsx [hunk 4]
```diff
@@ -30,10 +32,11 @@ function Navbar() {
   return (
     <header
       data-fs-navbar
+      data-fs-navbar-scroll={scrollDirection}
       className={`${styles.fsNavbar} layout__content-full`}
     >
-      <div className="layout__content" data-fs-navbar-header>
-        <section data-fs-navbar-row>
+      <section data-fs-navbar-header>
```
Comment (hellofanny) [inline src/components/common/Navbar/Navbar.tsx:38]: Actually, if I'd just for styling purpose, we could keep `<div>` 🤔

**Review by hellofanny — COMMENTED** [review]
AWwesomeee! 🤩 🤩 🤩 Nice work!

I've just noticed that the Navlinks are no longer displayed in the storybook. Can you check, pls? 👀 


|Before|After|
|-|-|
|image|image|

File: src/components/common/Navbar/Navbar.tsx (hunk 4 above)
Comment (lucasfp13) [reply src/components/common/Navbar/Navbar.tsx:39]: I think the intent here was just to prefer `section` instead `div` as the wrapper

**Comment by lucasfp13** [comment]
> a2d0ade

I noticed that too. Removing the `z-index` the component works normally 🤔

**Comment by lucasfp13** [comment]
> AWwesomeee! 🤩 🤩 🤩 Nice work!
> 
> I've just noticed that the Navlinks are no longer displayed in the storybook. Can you check, pls? 👀
> 
> Before	After
> image	image

I noticed that too. Removing the z-index the component works normally 🤔

**Review by lucasfp13 — APPROVED** [review]
After the fixes, LGTM!

**Review by hellofanny — APPROVED** [review]
I think [this](https://github.com/vtex-sites/nextjs.store/pull/191/commits/51c939a343be1349ea5484c64f673015c33e3b63) fix solved the previous issue. LGTM!

**Review by ArthurTriis1 — APPROVED** [review]

**Pull request merged**